src/app/pages/home/components/account-area.tsx

```diff
@@ -4,14 +4,14 @@ import { Stack, StackProps } from '@stacks/ui';
 
 import { CurrentAccountAvatar } from '@app/features/current-account/current-account-avatar';
 import { CurrentAccountName } from '@app/features/current-account/current-account-name';
-import { useCurrentAccountNativeSegwitAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+import { useCurrentAccountNativeSegwitIndexZeroSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
 import { useCurrentStacksAccount } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
 
 import { AccountTotalBalance } from '../../../components/account-total-balance';
 
 export const CurrentAccount = memo((props: StackProps) => {
   const currentAccount = useCurrentStacksAccount();
```

Comment:
I refactored this as I saw `useCurrentAccountNativeSegwitAddressIndexZero` is deprecated when I was reviewing the code

Reply:
Reading #3940 and I'll probably need to account for that here too as bitcoin address is sometimes `undefined`

Reply:
Let's revert this change as the behaviours are subtly different.

Reply:
Thanks, I'll remove that commit